### ISSUE TITLE
🐞 setPluginRegistrantCallback cannot find symbol Java

### ISSUE THREAD

**Issue by MiguelSOliveira** [vrtdev/flutter_workmanager#18]
I'm getting this error, am I missing something?
```
Application.java:12: error: cannot find symbol
    WorkmanagerPlugin.setPluginRegistrantCallback(this);
                     ^
  symbol:   method setPluginRegistrantCallback(Application)
  location: class WorkmanagerPlugin
1 error

FAILURE: Build failed with an exception.
```

Application.java 

```
package`package_name`;

import be.tramckrijte.workmanager.WorkmanagerPlugin;
import io.flutter.app.FlutterApplication;
import io.flutter.plugin.common.PluginRegistry;
import io.flutter.plugins.GeneratedPluginRegistrant;

public class Application extends FlutterApplication implements PluginRegistry.PluginRegistrantCallback {
  @Override
  public void onCreate() {
    super.onCreate();
    WorkmanagerPlugin.setPluginRegistrantCallback(this);
  }

  @Override
  public void registerWith(PluginRegistry registry) {
    GeneratedPluginRegistrant.registerWith(registry);
  }
}
```
I've added the custom application to android manifest.

I have the dependency: workmanager: ^0.0.11

**Comment by timrijckaert** [comment]
I see we have not exposed the `WorkManagerPlugin#setPluginRegistrantCallback` method for Java. 
Is there any reason you use Java here and not Kotlin?

Anyway I'll fix it.

**Comment by MiguelSOliveira** [comment]
No reason. It's just the flutter projects use java by default.. At least I haven't explicitely said I wanted Java, so I am guessing most people are the same?

**Comment by timrijckaert** [comment]
You are completely right!
I prepare a fix.

**Comment by timrijckaert** [comment]
The fix is now ready as version `0.0.12`.
Let me know if this fixes your issue.

**Comment by MiguelSOliveira** [comment]
Problem solved. Thank you for the quick fix and awesome lib !
